### PR TITLE
Delete Flow Topology

### PR DESCRIPTION
**Pull request by dpoltavets**
Closes: #3291

### CODE REVIEW

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java [hunk 1]
```diff
@@ -204,55 +203,27 @@
...
-                    required = false)
-            @RequestParam(value = "propagate", required = false) Optional<Boolean> propagate,
-            @ApiParam(value = "default: false. If true, will wait until poll timeout for validation.",
-                    required = false)
-            @RequestParam("verify") Optional<Boolean> verify) {
-
-        Boolean defaultPropagate = false;
-        Boolean defaultVerify = false;
-        return flowService.pushFlows(externalFlows, propagate.orElse(defaultPropagate), verify.orElse(defaultVerify));
+    public CompletableFuture<BatchResults> pushFlows() {
```
Comment (niksv) [inline src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java:214]: nit: add `@Deprecated` annotation and `@deprecated` java doc

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java [hunk 2]
```diff
@@ -204,55 +203,27 @@
...
-            @ApiParam(value = "default: false. If true, this will propagate rules to the switches.",
-                    required = false)
-            @RequestParam(value = "propagate", required = false) Optional<Boolean> propagate,
-            @ApiParam(value = "default: false. If true, will wait until poll timeout for validation.",
-                    required = false)
-            @RequestParam(value = "verify", required = false) Optional<Boolean> verify) {
-        Boolean defaultPropagate = false;
-        Boolean defaultVerify = false;
-        return flowService.unpushFlows(externalFlows, propagate.orElse(defaultPropagate), verify.orElse(defaultVerify));
+    public CompletableFuture<BatchResults> unpushFlows() {
```
Comment (niksv) [inline src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java:225]: same

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java [hunk 3]
```diff
@@ -166,23 +165,13 @@
...
     /**
      * Use this to push flows that may not be in the database / caches but they should be.
-     *
-     * @param externalFlows the list of flows to push.
-     * @param propagate     if true, the path/rules will be propagated to the switch
-     * @param verify        if true, we'll wait up to poll seconds to confirm if rules have been applied
      */
-    CompletableFuture<BatchResults> pushFlows(final List<FlowInfoData> externalFlows, Boolean propagate,
-                                              Boolean verify);
+    CompletableFuture<BatchResults> pushFlows();
```
Comment (niksv) [inline src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java:169]: same

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java [hunk 4]
```diff
@@ -166,23 +165,13 @@
...
     /**
      * Use this to unpush flows .. ie undo a push
-     *
-     * @param externalFlows the list of flows to unpush.
-     * @param propagate     if true, the path/rules will be propagated to the switch
-     * @param verify        if true, we'll wait up to poll seconds to confirm if rules have been applied
      */
-    CompletableFuture<BatchResults> unpushFlows(final List<FlowInfoData> externalFlows, Boolean propagate,
-                                                Boolean verify);
+    CompletableFuture<BatchResults> unpushFlows();
```
Comment (niksv) [inline src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java:174]: same

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java (hunk 1 above)
Comment (dpoltavets) [reply src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java:217]: Added.

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java (hunk 2 above)
Comment (dpoltavets) [reply src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java:231]: Added.

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java (hunk 3 above)
Comment (dpoltavets) [reply src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java:172]: Added.

File: src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java (hunk 4 above)
Comment (dpoltavets) [reply src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/FlowService.java:180]: Added.

File: src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/NetworkTopology.java [hunk 5]
```diff
@@ -137,7 +137,7 @@ private void inputSwitchManager(TopologyBuilder topology, int scaleFactor) {
 
     private void inputSpeakerRules(TopologyBuilder topology, int scaleFactor) {
         KafkaSpout<String, Message> spout = buildKafkaSpout(
-                topologyConfig.getFlowTopic(), ComponentId.INPUT_SPEAKER_RULES.toString());
+                topologyConfig.getSwitchManagerTopic(), ComponentId.INPUT_SPEAKER_RULES.toString());
```
[the remark this answers is not in the document]
Comment (dpoltavets) [reply src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/NetworkTopology.java:140]: Fixed.

File: src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java [hunk 6]
```diff
@@ -452,26 +396,6 @@ private void installServer42IngressFlow(final InstallServer42IngressFlow command
...
-     *
-     * @param message command message for flow installation
-     */
-    private void doProcessEgressFlow(final CommandMessage message, String replyToTopic, Destination replyDestination)
-            throws FlowCommandException {
-        InstallEgressFlow command = (InstallEgressFlow) message.getData();
-        logger.info("Installing egress flow '{}' on switch '{}'", command.getId(), command.getSwitchId());
-
-        try {
-            installEgressFlow(command);
```
Comment (surabujin) [inline src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java:466]: What about `installEgressFlow`, `installEgressFlow` and `installOneSwitchFlow` methods? What about deeper handling triggered from these methods?

File: src-java/flow-topology/flow-storm-topology/src/test/java/org/openkilda/messaging/command/flow/AbstractSerializerTest.java [hunk 7]
```diff
@@ -1,640 +0,0 @@
...
-
-        FlowsResponse resultData = (FlowsResponse) resultInfo.getData();
-        System.out.println(resultData);
-        assertEquals(data, resultData);
-        assertEquals(data.hashCode(), resultData.hashCode());
-        assertEquals(Collections.singletonList(flowModel.getFlowId()).hashCode(), resultData.getFlowIds().hashCode());
-    }
-
-    @Test
-    public void eventIslInfoTest() throws IOException, ClassNotFoundException {
```
[the remark this answers is not in the document]
Comment (dpoltavets) [reply src-java/flow-topology/flow-storm-topology/src/test/java/org/openkilda/messaging/command/flow/AbstractSerializerTest.java:417]: These tests have long been ignored and do not test anything.

File: src-java/flow-topology/flow-storm-topology/src/test/java/org/openkilda/wfm/topology/utils/LoggerContextInitializerTest.java [hunk 8]
```diff
@@ -1,91 +0,0 @@
...
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import org.apache.storm.tuple.Fields;
-import org.apache.storm.tuple.Tuple;
-import org.junit.Test;
-
-import java.util.Optional;
-import java.util.UUID;
-
-public class LoggerContextInitializerTest {
```
[the remark this answers is not in the document]
Comment (dpoltavets) [reply src-java/flow-topology/flow-storm-topology/src/test/java/org/openkilda/wfm/topology/utils/LoggerContextInitializerTest.java]: These tests have been restored.

File: src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java (hunk 6 above)
Comment (dpoltavets) [reply src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java:466]: These methods have been removed.

**Pull request merged**